test/pubtator/pubtator.js

```diff
@@ -77,39 +78,100 @@ describe('pubtator', function () {
           expect(inAbstract.length).to.equal(uniqInAbstract.length);
         });
 
-        it(`Should be all an ORGANISM hint`, function () {
-          const isOrganism = (h) => h.type === HINT_TYPE.ORGANISM;
-          expect(hints.every(isOrganism)).to.be.true;
-        });
-
         it('Should aggregate organism hints correctly', function () {
           const aggregateCounts = hints.reduce((acc, hint) => {
             const key = `${hint._xref.id}_${hint.section}`;
```

Comment:
This avoids the use of the model, which I believe defeats the purposes of the model.